ia_2048.py

```diff
@@ -0,0 +1,27 @@
+import move_2048
+
+def set_direction(grid, tour):
+    dic_move = {0: "g", 1: "d", 2: "h", 3: "b"}
```

Comment:
this kind of dict looks like a list to me: you have all the keys from 0 to 3, you do not need a dict, you need a list